src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -254,6 +254,13 @@ namespace winrt::TerminalApp::implementation
         _layoutUpdatedRevoker = _tabContent.LayoutUpdated(winrt::auto_revoke, { this, &TerminalPage::_OnFirstLayout });
 
         _isAlwaysOnTop = _settings.GlobalSettings().AlwaysOnTop();
+
+        // Setup mouse vanish attributes
+        SystemParametersInfoW(SPI_GETMOUSEVANISH, 0, &_shouldMouseVanish, false);
+
+        // Store cursor, so we can restore it, e.g., after mouse vanishing
+        // (we'll need to adapt this logic once we make cursor context aware)
+        _defaultPointerCursor = CoreWindow::GetForCurrentThread().PointerCursor();
```

Comment:
qq: is this true for a pen display in pen cursor mode?

Reply:
Sincerely speaking - I didn't even consider this.. :sad:

The initial digging in the documentation didn't provide too much value.. I will look more..

Of course, it could be really good to test it as well, but all my pens at home are with real ink... (and the offices are locked down).

Is there some kind of emulator I could play with? Alternatively, I am ordering a surface and sending ms expense to your cc 😜.

Reply:
Just tested it on my SB3. The mouse cursor position is handled independently of the stylus cursor position. So the following behavior occurs:
- Move mouse. Typing hides mouse cursor. Move/Interact w/ stylus. Moving mouse starts at original mouse position (not stylus position)
- Move mouse. Moving stylus hides mouse cursor.

I think this is fine. Word keeps the mouse cursor position independent from the stylus position. However, VS does not (probably because the stylus is just another manifestation of the mouse cursor). It's honestly a small enough thing that we should probably move past?

Reply:
@DHowett - it's been quite a while so bumping :blush:

Reply:
Got it. Thanks!